tools/import-strings.sh

```diff
@@ -33,14 +33,27 @@ fi
 echo "[*] Cloning mozilla-l10n/focusios-l10n"
 git clone https://github.com/mozilla-l10n/focusios-l10n.git
 
+echo "[*] Cloning mozilla-mobile/LocalizationTools"
+[ -d tools/Localizations ] && rm -rf tools/Localizations
+git clone https://github.com/mozilla-mobile/LocalizationTools.git tools/Localizations
+
 echo "\n\n[*] Building tools/Localizations"
 (cd tools/Localizations && swift build)
 
+echo "[*] Replacing firefox with focus in swift task files"
+sed -i '' 's/firefox-ios.xliff/focus-ios.xliff/g' tools/Localizations/Sources/LocalizationTools/tasks/*.swift
+
+echo "[*] Removing es-ES locale mapping from swift import task"
+sed -i '' '/es-ES/d' tools/Localizations/Sources/LocalizationTools/tasks/ImportTask.swift
+
+echo "[*] Removing WidgetKit/en-US.lproj/WidgetIntents.strings from swift import task"
+# Match all text between a line containing 'ShortcutItemTitleQRCode' to ']' and delete them
+sed -ri '' '/ShortcutItemTitleQRCode/,/\]/{/ShortcutItemTitleQRCode/!{/\]/!d;};}' tools/Localizations/Sources/LocalizationTools/tasks/ImportTask.swift
+
 echo "\n\n[*] Importing Strings - takes a minute. (output in import-strings.log)"
```

Comment:
ubernit: probably want to remove the two line breaks before this message, or add them to the rest of the output